CONTRIBUTING.md

```diff
@@ -1,14 +1,109 @@
 
-Feel free to make changes to the template files or the document files. The supporting docs are located in their respective directories, and the API docs are located under the `_includes` directory.
+>[!NOTE]
+> This is not the repo for Express JS. To contribute to the _[Express JS framework](https://github.com/expressjs/express)_ itself, [check this out](#were-the-express-js-frameworks-documenntation-team---not-the-express-js-framework).
 
-Content on this site is licensed under a Creative Commons Attribution-ShareAlike 3.0 United States License.  See https://creativecommons.org/licenses/by-sa/3.0/us/ for a layman's summary; 
-See [LICENSE.md](LICENSE.md) for the full license.
+If you want to contribute to [https://github.com/expressjs/expressjs.com](https://github.com/expressjs/expressjs.com), you're in the right place. 
 
+
+Whether you're a first time first time contributor or you're just looking for a refresher, visit our [Getting Started](#contributors-guide-to-getting-started) guide below to get up and running.
+
+#### Want to help but need some ideas? These are some of our most typical issue types:
+
+1.  **Website Related**: 
+If you see anything on the site that could use a tune-up, think about how to fix it.
+
+    - display or screen sizing problems
+    - mobile responsiveness issues
+    - missing or broken accessibility features 
+    - website outages
+    - broken links
+    - page structure or user interface enhancements
+
+
+2. **Content Related**: spelling errors, incorrect/outdated Express JS documentation, adding missing content.
+    - We love any reports of typos. If you see one, fixing it is a great way to help.
+
+
+3. **Translation Related**: spelling errors, incorrect/poorly translated words, adding new full site translations.
+> [!IMPORTANT]
+> All translation submissions are currently paused. See this [notice](#notice-we-have-paused-all-translation-contributions) for more information.
+
+   - If you want to do a full site translation, or just translate a single page, there's a section specifically related to [translations](#contributing-translations) below.
+
+#### Want to work on a backlog issue?
+
+We often have bugs or enhancements that need work. You can find these under our repo's [Issues tab](https://github.com/expressjs/expressjs.com/issues). Check out the tags to find something that's a good match for you.
+
+#### Have an idea? Found a bug?
+
+If you've found a bug or a typo, or if you have an idea for an enhancement, you can:
+- Submit a [new issue](https://github.com/expressjs/expressjs.com/issues/new/choose) on our repo. Do this for larger proposals, or if you'd like to discuss or get feedback first. 
+- Make a [Github pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). If you have already done work and it's ready to go, feel free to send it our way.
+
+## Contributor's Guide to Getting Started
```

Comment:
This wording is confusing, since there's a "Getting started" section of the docs.... Is this about contributing to that? No--so change to just "Getting started".